server/src/main/java/com/linecorp/centraldogma/server/internal/metadata/MetadataRepository.java

```diff
@@ -42,14 +42,14 @@
 import com.linecorp.centraldogma.server.internal.storage.project.ProjectManager;
 import com.linecorp.centraldogma.server.internal.storage.repository.Repository;
 
-final class RepositoryUtil<T> {
+final class MetadataRepository<T> {
```

Comment:
How about `MetadataAccess`? It's not a `Repository`.

Reply:
How about some other name such as `RepositoySupport`? Because it's not for metadata only.